exchanges/lbank/lbank_wrapper.go

```diff
@@ -0,0 +1,546 @@
+package lbank
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/orderbook"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/ticker"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/websocket/wshandler"
+	log "github.com/thrasher-corp/gocryptotrader/logger"
+)
+
+// Start starts the Lbank go routine
+func (l *Lbank) Start(wg *sync.WaitGroup) {
+	wg.Add(1)
+	go func() {
+		l.Run()
+		wg.Done()
+	}()
+}
+
+// Run implements the Lbank wrapper
+func (l *Lbank) Run() {
+	if l.Verbose {
+		log.Debugf("%s Websocket: %s. (url: %s).\n", l.GetName(), common.IsEnabled(l.Websocket.IsEnabled()), l.Websocket.GetWebsocketURL())
+		log.Debugf("%s polling delay: %ds.\n", l.GetName(), l.RESTPollingDelay)
+		log.Debugf("%s %d currencies enabled: %s.\n", l.GetName(), len(l.EnabledPairs), l.EnabledPairs)
+	}
+	exchangeCurrencies, err := l.GetCurrencyPairs()
+	if err != nil {
+		log.Errorf("%s Failed to get available symbols.\n", l.GetName())
+	} else {
+		var newExchangeCurrencies currency.Pairs
+		for _, p := range exchangeCurrencies {
+			newExchangeCurrencies = append(newExchangeCurrencies,
+				currency.NewPairFromString(p))
+		}
+		err = l.UpdateCurrencies(newExchangeCurrencies, false, true)
+		if err != nil {
+			log.Errorf("%s Failed to update available currencies %s.\n", l.GetName(), err)
+		}
+	}
+}
+
+// UpdateTicker updates and returns the ticker for a currency pair
+func (l *Lbank) UpdateTicker(p currency.Pair, assetType string) (ticker.Price, error) {
+	var tickerPrice ticker.Price
+	tickerInfo, err := l.GetTicker(p.String())
```

Comment:
It's best to use exchange.FormatExchangeCurrency here and for the other wrappers so it will take care of the pair formatting in case someone puts something unexpected in